src/app/clusters/general-commissioning-server/general-commissioning-server.cpp

```diff
@@ -258,12 +258,37 @@ bool emberAfGeneralCommissioningClusterSetRegulatoryConfigCallback(app::CommandH
 {
     MATTER_TRACE_EVENT_SCOPE("SetRegulatoryConfig", "GeneralCommissioning");
     DeviceControlServer * server = &DeviceLayer::DeviceControlServer::DeviceControlSvr();
+    Commands::SetRegulatoryConfigResponse::Type response;
+    uint8_t location = to_underlying(commandData.newRegulatoryConfig);
+
+    if (location > to_underlying(chip::app::Clusters::GeneralCommissioning::RegulatoryLocationType::kIndoorOutdoor))
```

Comment:
```suggestion
    if (location > to_underlying(RegulatoryLocationType::kIndoorOutdoor))
```
That said, you could also just `if (commandData.newRegulatoryConfig > RegulatoryLocationType::kIndoorOutdoor)` and then only declare `location` in the block that needs it.  Or cast the result of `GetLocationCapability` to `RegulatoryLocationType` and not have any `to_underlying` around at all.